XIVSlothCombo/Combos/CustomComboPreset.cs

```diff
@@ -1863,12 +1863,16 @@ public enum CustomComboPreset
         NIN_HuraijinArmorCrush = 10064,
 
         [ParentCombo(CustomComboPreset.NIN_ST_AdvancedMode_RangedUptime)]
-        [CustomComboInfo("Raiton Uptime Option", "Adds Raiton as an uptime feature.", NIN.JobID)]
-        NIN_ST_AdvancedMode_Raiton_Uptime = 10065,
+        [CustomComboInfo("Phantom Kamaitachi Uptime Option", "Adds Phantom Kamaitachi as an uptime feature.", NIN.JobID)]
```

Comment:
Please use my original numbering scheme.
Raiton Uptime -> 10065
Phantom Uptime -> 10066,
Suiton Uptime -> 10067

Reply:
I'm happy to do that, but is there a specific reason for this? Having it ordered in priority terms and then having the IDs updated to match made sense to me. Do you want me to restore the previous order too?

Reply:
We never change numbers once they're live, unless it's a full reset which this does not call for. These dictate people's settings, arbitrarily changing numbers isn't a fun time to explain to people why stuff is turned off and something else is turned on.

Reply:
And these aren't live so now is the time to change them if we want to, no?